packages/edit-site/src/components/global-styles/style-variations-container.js

```diff
@@ -41,10 +41,28 @@ function Variation( { variation } ) {
 	}, [ variation, base ] );
 
 	const selectVariation = () => {
+		const blockCSS = {};
+		if ( user?.styles?.blocks ) {
+			Object.keys( user.styles.blocks ).forEach( ( blockName ) => {
+				if ( user.styles.blocks[ blockName ].css ) {
+					blockCSS[ blockName ] = {
+						css: user.styles.blocks[ blockName ].css,
+					};
+				}
+			} );
+		}
+
 		setUserConfig( () => {
 			return {
 				settings: variation.settings,
-				styles: variation.styles,
+				styles: {
+					...variation.styles,
+					...( user?.styles?.css ? { css: user.styles.css } : {} ),
```

Comment:
I need to tidy the following up as this won't merge properly - I need to also check if a them variation can in fact include block styles.